pkg/authorization/workspace_content_authorizer.go

```diff
@@ -72,12 +72,18 @@ func (a *OrgWorkspaceAuthorizer) Authorize(ctx context.Context, attr authorizer.
 	if cluster == nil || cluster.Name == "" {
 		return authorizer.DecisionNoOpinion, "", nil
 	}
-	orgWorkspace, workspace, err := helper.ParseLogicalClusterName(cluster.Name)
+
+	parentClusterName, err := helper.ParentClusterName(cluster.Name)
```

Comment:
suggestion: if "parent cluster == org workspace", then let's call it "org cluster" or the like to avoid confusion.

Reply:
it can be root as well, the parent of an org cluster :)